lldb/source/Core/Module.cpp

```diff
@@ -681,13 +681,13 @@ Module::LookupInfo::LookupInfo(ConstString name,
            language == eLanguageTypeSwift) &&
           swift_method.IsValid())
         basename = swift_method.GetBasename();
-      else if ((language == eLanguageTypeUnknown ||
-                Language::LanguageIsCPlusPlus(language) ||
-                Language::LanguageIsC(language) ||
-                language == eLanguageTypeObjC_plus_plus) &&
-               cpp_method.IsValid())
-        basename = cpp_method.GetBasename();
 #endif // LLDB_ENABLE_SWIFT
+      if ((language == eLanguageTypeUnknown ||
+              Language::LanguageIsCPlusPlus(language) ||
+              Language::LanguageIsC(language) ||
+              language == eLanguageTypeObjC_plus_plus) &&
```

Comment:
May be you could use this ?

```suggestion
              Language::LanguageIsCFamily(language) &&
```